molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/settings/base.py

```diff
@@ -213,7 +213,7 @@
 # Internationalization
 # https://docs.djangoproject.com/en/1.7/topics/i18n/
 LANGUAGE_CODE = environ.get('LANGUAGE_CODE', 'en')
-TIME_ZONE = 'Africa/Johannesburg'
+TIME_ZONE = 'UTC'
```

Comment:
do we need to change this? will mean current schedules will change if they do not set a different timezone?

Reply:
This is the cookiecutter so my understanding is that it's just used for new sites with `molo scaffold`, is that right?

For existing sites the migration ensures that we use the value that's currently set in `TIME_ZONE` which for most should be SAST I think.

Reply:
very true 👍